### DISCUSSION TITLE
Add "Sensitive Sites" Short-lived Cookies?

### DISCUSSION THREAD

**Discussion by lostgit** [Cookie-AutoDelete/Cookie-AutoDelete#1177] [Ideas / Feature Requests]
Would be great to have another list beside WHITE and GREY.  For usability, I tend to allow cookies to lie around for a good part of the day for most sites (long timeout), but for financial sites it would be nice for CAD to clean them up promptly and more frequently.

Maybe a GREEN list? Sites listed here will have cookies expired after their tabs are closed or after XYZ seconds (a separate setting than the main one, so we can set a shorter value).  

Think financial sites and other sites we want to be cleaned up promptly and often.

**Comment by kennethtran93** [comment]
The custom delay per list isn't on our radar yet.  Our current workflow is after the global delay, we fetch all cookies (this is always the first step) and then run a check for each cookie against the lists. I don't think it would be feasible to introduce another delay again after the check as we would need to query again after the delay to see if that cookie still exists before finally deleting it. Not to mention if the configuration have changed since the secondary delay such that the delay cookie shouldn't be deleted anymore, we'd need to process the lists one more time.

I have on the radar, however, self-customized lists and the ability for cookies to be matched to multiple lists, among other items.
